src/libOpenImageIO/imagebufalgo_compare.cpp

```diff
@@ -21,6 +21,8 @@
 
 #include "imageio_pvt.h"
 
+using namespace std;
+
```

Comment:
I'm not super crazy about blanket 'using namespace std'. Can we just put `std::` in front of the places that need it?

Reply:
Thanks for the review!
There were just a lot more in this file :-) I've used prefixes now.

Reply:
Oops, I think you need to re-clangformat that one file now.